.changes/unreleased/Under the Hood-20220914-150058.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Under the Hood
```

Comment:
Not sure where this falls in terms of changelog type?

Reply:
Since we're deprecating instead of erroring now, this won't be breaking.  I'll leave as `Under The Hood` unless you disagree.